src/gui/accountmanager.cpp

```diff
@@ -78,40 +77,45 @@ bool AccountManager::restore()
 bool AccountManager::restoreFromLegacySettings()
 {
     qCInfo(lcAccountManager) << "Migrate: restoreFromLegacySettings, checking settings group"
-             << Theme::instance()->appName();
+                             << Theme::instance()->appName();
```

Comment:
@jturcotte Actually, clang-format seems to align `<<` no matter what that setting is. I recommend looking at the patchset for both cases. (alwaysbreak reformats everything to be on one line if columnlimit is disabled) I'm currently leaning towards DontAlign.